examples_tests/17.SimpleBulletIntegration/Camera.hpp

```diff
@@ -163,6 +163,46 @@ class Camera {
 				}
 			}
 
+			if(ev.type == ui::SMouseEvent::EET_MOVEMENT && mouseDown) {
+				float rotateSpeed = 0.2f;
+				core::vectorSIMDf pos; pos.set(getPosition());
+				core::vectorSIMDf target = getTarget() - pos;
+				core::vector3df relativeRotation = target.getAsVector3df().getHorizontalAngle();
+				relativeRotation.X -= ev.movementEvent.movementY * rotateSpeed * -1.0f;
+				float tmpYRot = ev.movementEvent.movementX * rotateSpeed * -1.0f;
+				if (leftHanded)
+					relativeRotation.Y -= tmpYRot;
+				else
+					relativeRotation.Y += tmpYRot;
+
+				constexpr float MaxVerticalAngle = 88.0f;
```

Comment:
dont take after the old irrlicht contributors, do your math in radians not degrees